<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>221</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Красно-черное дерево:</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_deleteKey">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить по ключу</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_key">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_insert">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Вставить ключ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
